skglm/solvers/prox_newton.py

```diff
@@ -42,7 +42,7 @@ class ProxNewton(BaseSolver):
     """
 
     def __init__(self, p0=10, max_iter=20, max_pn_iter=1000, tol=1e-4,
-                 fit_intercept=True, warm_start=False, verbose=0):
+                 fit_intercept=False, warm_start=False, verbose=0):
```

Comment:
changing default has a silent impact on users, we try to avoid it as much as possible (though our userbase may not be so large, let's take good habits from the start)

Reply:
Yes, I totally agree with you. I will revert it.
(I made such a change to pass tests)

Reply:
perfect :)